src/callkit.cs

```diff
@@ -135,7 +135,7 @@ public enum CXHandleType : long {
 		EmailAddress = 3,
 	}
 
-	[iOS (10, 0), NoMac]
+	[iOS (10, 0), NoMac, Watch (9,0), MacCatalyst (13,0)]
```

Comment:
Mac Catalyst 13.0? Or should that be 16.0?

Reply:
Is the default, I made a mistake, will fix those.